Doc/library/concurrent.interpreters.rst

```diff
@@ -13,17 +13,26 @@
 
 --------------
 
-
-Introduction
-------------
-
 The :mod:`!concurrent.interpreters` module constructs higher-level
 interfaces on top of the lower level :mod:`!_interpreters` module.
 
-.. XXX Add references to the upcoming HOWTO docs in the seealso block.
+The module is primarily meant to provide a basic API for managing
+interpreters (AKA "subinterpreters") and running things in them.
+Running mostly involves switching to an interpreter (in the current
+thread) and calling a function in that execution context.
```

Comment:
I'd like a reference to the "Runtime Components" section after #135944 is landed, regarding the OS thread?

Reply:
Places like these are a large part of why I opened that issue. 😄